src/style.css

```diff
@@ -1,3 +1,67 @@
+html {
+  font-family: "Roboto", sans-serif;
+  --text-color: #00acdc;
+  --link-heigh: 60px;
```

Comment:
it's better not hardcode height for elements with content inside. You can just use padding: 23px 0; for your links